src/Storages/MergeTree/MutateTask.cpp

```diff
@@ -1457,12 +1513,38 @@ class MutateAllPartColumnsTask : public IExecutableTask
         bool is_full_part_storage = isFullPartStorage(ctx->new_data_part->getDataPartStorage());
         const auto & indices = ctx->metadata_snapshot->getSecondaryIndices();
 
+        bool secondary_indices_on_columns_alter_modify_drop = false;
```

Comment:
It is not clear from the variable name what the variables mean.

Can we not use a single variable of type `SecondaryIndicesOnColumnsAlterModify` (= the same type as in Core/Settings.cpp)?